ansible/roles/davros/tasks/main.yml

```diff
@@ -12,3 +12,8 @@
   template:
     src: nginx/davros.conf
     dest: /etc/nginx/sites-enabled/davros.conf
+
+- name: Copy the initialization script
+  copy: src=initialize.d/davros.sh
```

Comment:
Let's stick to yaml syntax, i.e. no `src=` but instead `src:`

Reply:
Separate PR for https://github.com/liquidinvestigations/setup/issues/11. I'm sure there's a script somewhere on the internets that lets us automate this.